core/src/scss/elements/input/_radio-checkbox.scss

```diff
@@ -0,0 +1,85 @@
+@charset "UTF-8";
+
+// Custom checkboxes and radio buttons.
+[type='checkbox'],
+[type='radio'] {
+  // The actual input element is only visible to screen readers, because
+  // all visual styling is done via the label.
+  @include hide-visually;
+
+  .lt-ie9 & {
+    border: 0;
+    float: left;
+    margin: 0.4em 0.4em 0 0;
+    position: static;
+    width: auto;
+  }
+}
+
+[type='checkbox'] + label,
+[type='radio'] + label {
+  cursor: pointer;
+  font-weight: 400;
+  margin-bottom: 0.5em;
+}
+
+[type='checkbox'] + label::before,
+[type='radio'] + label::before {
+  background: $color-white;
+  border-radius: $border-radius;
+  box-shadow: 0 0 0 1px $color-driftwood;
+  content: '\a0';
+  display: inline-block;
+  height: 1.8rem;
+  line-height: 1.8rem;
+  margin-right: 0.6em;
+  text-indent: 0.15em;
+  vertical-align: middle;
+
+  // Target IE 11 and below to vertically center inputs.
```

Comment:
I don't see why the width would be for IE 11 and below for vertical center. Can you elaborate on this comment?

Reply:
Again, not my code - I just copied and pasted them into their own file 😂 . The radio button/ checkboxes actually need a ton of work. There is a lot of questionable stuff there.